src/walk-path.js

```diff
@@ -6,21 +6,9 @@ import './walk-path.css';
 class WalkPath extends React.Component {
   constructor() {
```

Comment:
The constructor can now go away completely (the linter complains about a superfluous constructor).